xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.cpp

```diff
@@ -120,6 +147,78 @@ enum AVPixelFormat CDVDVideoCodecDRMPRIME::GetFormat(struct AVCodecContext* avct
   return AV_PIX_FMT_NONE;
 }
 
+static void ReleaseBuffer(void* opaque, uint8_t* data)
+{
+  CVideoBufferDMA* buffer = static_cast<CVideoBufferDMA*>(opaque);
+  buffer->Release();
+}
+
+static void AlignedSize(AVCodecContext* avctx, int& width, int& height)
```

Comment:
General question as these positions are not my area :smile:.

Looked here and in some other places, static functions that are not in a namespace or class.
These are passed to ffmpeg, but for security and our guidelines, it might be good to put them in at least one namespace?

Reply:
That's a great point. I've move the static functions to the anonymous namespace